protocols/smart-transfer-protocol.openapi.yaml

```diff
@@ -0,0 +1,228 @@
+openapi: 3.0.3
+
+info:
+  title: ODF Smart Transfer Protocol (HTTP endpoints part)
+  description:
+    Open Data Fabric (ODF) extended protocol for dataset exchange between remote parties.
+    Defines REST API endpoints that enable simple operation and upgrading to asynchronous API protocol.
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: 0.1.0
+
+components:
+
+  parameters:
+
+    Reference:
+      in: path
+      name: reference
+      description: Name of the reference
+      example: head
+      required: true
+      schema:
+        type: string
+        enum: [head]
+
+    BlockHash:
+      in: path
+      name: blockHash
+      description: Hash of the requested block
+      example: zW1qJPmDvBxGS9GeC7PFseSCy7koHjvurUmisf1VWscY3AX
+      required: true
+      schema:
+        type: string
+        format: multihash
+
+    ObjectPhysicalHash:
+      in: path
+      name: physicalHash
+      description: Physical hash of the requested object file
+      example: zW1diFMSn97sDG4WMMKZ7pvM7vVenC5ytAesQK7V3qqALPv
+      required: true
+      schema:
+        type: string
+        format: multihash
+
+    UpgradeToWebSocket:
+      in: header
+      name: Upgrade
+      schema:
+        type: string
+        enum: [websocket]
+      required: true
+
+    ConnectionUpgrade:
+      in: header
+      name: Connection
+      schema:
+        type: string
+        enum: [Upgrade]
+      required: true
+
+    WebSocketProtocol:
+      in: header
+      name: Sec-WebSocket-Protocol
+      schema:
+        type: string
+        enum: [open-data-fabric]
```

Comment:
Since `Sec-WebSocket-Version` seems to refer to the version of websocket protocol itself, should we include ODF protocol version in this header?

e.g. `odf/smart-transfer-protocol/v1`

Reply:
Ok. Created a schema, since this is shared between headers in requests and respopnse